modules/gathering-data-specific-features.adoc

```diff
@@ -63,6 +63,9 @@ endif::openshift-dedicated[]
 |`registry.redhat.io/workload-availability/self-node-remediation-must-gather-rhel8:v<installed-version-SNR>`
 |Data collection for the Self Node Remediation (SNR) Operator and the Node Health Check (NHC) Operator.
 
+|`registry.redhat.io/numaresources/numaresources-must-gather-rhel9:v<installed-version-NRO>`
```

Comment:
User-replaceable values should be lowercase and underscore-separated. I see there is a mix of formatting in this table already, but would be good to only add new ones following the convention.
```suggestion
|`registry.redhat.io/numaresources/numaresources-must-gather-rhel9:v<installed_version_nro>`
```